monitoring/src/main/scala/akkaviz/events/ActorSystems.scala

```diff
@@ -0,0 +1,21 @@
+package akkaviz.events
+
+import akka.actor.ActorSystem
+
+import scala.ref.WeakReference
+
+object ActorSystems {
+
+  private val _systems = scala.collection.mutable.Map[String, WeakReference[ActorSystem]]()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2106955461/issues/source?bid=3121282&fileBranchId=3144103#l9)